RH_CC110.cpp

```diff
@@ -189,17 +189,29 @@ void RH_CC110::handleInterrupt()
 // These are low level functions that call the interrupt handler for the correct
 // instance of RH_CC110.
 // 3 interrupts allows us to have 3 different devices
+#if defined(ESP8266)
+ICACHE_RAM_ATTR void RH_CC110::isr0()
+#else
 void RH_CC110::isr0()
+#endif
 {
     if (_deviceForInterrupt[0])
 	_deviceForInterrupt[0]->handleInterrupt();
 }
+#if defined(ESP8266)
+ICACHE_RAM_ATTR void RH_CC110::isr1()
+#else
 void RH_CC110::isr1()
+#endif
 {
     if (_deviceForInterrupt[1])
 	_deviceForInterrupt[1]->handleInterrupt();
 }
+#if defined(ESP8266)
+ICACHE_RAM_ATTR void RH_CC110::isr2()
```

Comment:
you just need to ifdef the ICACHE word right? or you can #define ICACHE_RAM_ATTR for non-ESP8266 parts to nothing?